### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by VicPeralta**
# Implements Desktop version

### CODE REVIEW

**Comment by VicPeralta** [comment]
# Portfolio: desktop version
This branch implements the desktop version of the portfolio project.
- No linters errors
- Use GitHub flow
- Looks like Figma design
- Uses media queries for mobile version (width less than 640px), tablet version (between 641px and 991px), desktop version(between 992px and 1100px) and wide desktop version (wider then 1101px)
- Uses CSS grid and Flexbox
- Replicates the Figma design as much as possible

File: styles.css [hunk 1]
```diff
@@ -8,6 +8,17 @@ body {
   font-family: "Poppins", Arial, Helvetica, sans-serif;
 }
```
Comment (medaminedev66) [inline styles.css:10]: [OPTIONAL]
I would suggest adding `margin: 0` to the `*`, so we are setting all elements to have zero margins, thus making them look the same in all browsers. Notice that you have a small margin on the left, right, and bottom of the contact section.
```CSS
* {
  margin: 0;
  box-sizing: border-box;
}
```

**Review by medaminedev66 — COMMENTED** [review]
Hi @VicPeralta and @Have-Samuel, 

## Status: Approved

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (VicPeralta) [reply styles.css:10]: Thanks for the suggestion, it is a good one, I'll change it.

**Pull request merged**